robyn/cli.py

```diff
@@ -129,4 +129,7 @@ def run():
         print("Starting dev server...")
         start_dev_server(config, config.file_path)
     else:
-        start_app_normally(config)
+        try:
+            start_app_normally(config)
+        except KeyboardInterrupt:
+            pass
```

Comment:
@VishnuSanal , could you add a comment here?

Reply:
I would like to suggest some comments that explicitly suggest why is this a good solution? Or atleast `crashes on pressing ctrl+c`